scripts/ci/jenkins/simctl.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Ensure the correct CoreSimulatorService is loaded.
+
+xcrun simctl help >/dev/null 2>&1
+xcrun simctl help >/dev/null 2>&1
+xcrun simctl help >/dev/null 2>&1
```

Comment:
Replace with this:

https://github.com/calabash/DeviceAgent.iOS/blob/develop/bin/simctl.sh